demos/BMW_demo/bmw_open.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Share the given share to all the parties in the jiff instance.
+ * @function jiff_broadcast
+ * @ignore
+ * @param {module:jiff-client~JIFFClient} jiff - the jiff client instance.
+ * @param {module:jiff-client~JIFFClient#SecretShare} share - the share.
+ * @param {Array} parties - the parties to broadcast the share to.
+ * @param {number|string} op_id - a unique operation id, used to tag outgoing messages.
+ *
+ */
+
+/*
+
+function ooo(jiff,ls) {
+  var re=ls[1];
+  for (var i=2;i<=Object.keys(ls).length;i++) {
+    re=re^ls[i];
+
+  }
+  return re;
+
+}
+
+function recon_ls(jiff,ls) {
+  var re=ls[0];
+  for (var i=1;i<Object.keys(ls).length;i++) {
+    re=re^ls[i];
+
+  }
+  return re;
+
+}
+*/
+function recon_ls(jiff,ls) {
+  var re=ls[0];
+  for (var i=1;i<Object.keys(ls).length;i++) {
+    re=re^ls[i];
+
+  }
+  return re;
+
+
+}
+
+function bmw_reconstruct(jiff,shares) {
+  //console.log("inhelp");
+  //console.log(shares);
+  var ls=[];
+  for (let [key] of Object.keys(shares)) {
+    ls.push(shares[key]['value']);
+
+  }
+  //console.log(ls);
+  return recon_ls(jiff,ls);
+
+
+
+}
+
+var jiff_broadcast = function (jiff, share, parties, op_id) {
+  for (var index = 0; index < parties.length; index++) {
+    var i = parties[index]; // Party id
+    if (i === jiff.id) {
+      jiff.handlers.receive_open({ party_id: i, share: share.value, op_id: op_id, Zp: share.Zp });
+      continue;
+    }
+
+    // encrypt, sign and send
+    var msg = {party_id: i, share: share.value, op_id: op_id, Zp: share.Zp};
+    msg = jiff.hooks.execute_array_hooks('beforeOperation', [jiff, 'open', msg], 2);
+    msg['share'] = jiff.hooks.encryptSign(jiff, msg['share'].toString(), jiff.keymap[msg['party_id']], jiff.secret_key);
+    jiff.socket.safe_emit('open', JSON.stringify(msg));
+  }
+};
+
+module.exports = {
+  /**
+   * Open up the given share to the participating parties.
```

Comment:
Should update the documentation to reflect the name, parameters, and result of this function (same for all functions we've defined). This will make integrating into the jiff library easier later on since we will already have documentation.